### ISSUE TITLE
missing support for security type `OpenID Connect Discovery` (openIdConnect)

### ISSUE THREAD

**Issue by joenio** [byu-oit/openapi-enforcer#115]
```sh
npm test test/openapi-syntax.test.js 

> risis-rcf-risor@0.1.0 test /home/joenio/src/lisis/risis-rcf-risor
> jest "test/openapi-syntax.test.js"

 FAIL  test/openapi-syntax.test.js
  OpenAPI document
    ✕ openapi.yaml file is valid (129 ms)

  ● OpenAPI document › openapi.yaml file is valid

    One or more errors exist in the OpenApi definition
      at: security > 0 > OpenID > OpenID
        Security requirement for openIdConnect value must be an empty array
```

But OpenAPI specs says that `OpenID` must not be empty array, it must be filled by the scopes in a array format.
* https://swagger.io/docs/specification/authentication/openid-connect-discovery/

**Comment by Gi60s** [comment]
Thanks for sharing this issue. I'll look into it and hopefully have a fix tomorrow.

**Comment by joenio** [comment]
Thank you very much @Gi60s

**Comment by Gi60s** [comment]
The fix has been published in version `1.14.1`. Thanks again for the issue.
